network-api/networkapi/templates/pages/libraries/detail_page.html

```diff
@@ -51,7 +51,7 @@ <h2 class="tw-h3-heading tw-my-12">{% trans "Overview" context "Library detail p
         </div>
 
         <div id="authors-and-collaborators" class="medium:tw-col-start-3 medium:tw-col-end-4 tw-mt-12 medium:tw-mt-24">
-            {% if page.localized_authors %}
+            {% if page.authors %}
```

Comment:
Updated to use `authors`, as `localized_authors` will only return "related author" relationships that are stored in the database. Authors will return existing relations from the database, and any pending updates to the list that are stored in CMS memory.